node/core/approval-voting/Cargo.toml

```diff
@@ -32,6 +32,7 @@ sp-application-crypto = { git = "https://github.com/paritytech/substrate", branc
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 
 [dev-dependencies]
+async-std = "1.6.5"
```

Comment:
seems it's only used for `async_std::task::sleep` which is an overkill, we can use `futures_timer::Delay` instead

Reply:
Ah yes, I forgot to remove async-std. Great catch, thank you!